src/Microsoft.ComponentDetection.Detectors/maven/MvnCliComponentDetector.cs

```diff
@@ -33,19 +36,24 @@ public class MvnCliComponentDetector : FileComponentDetector
 
     public override string Id => "MvnCli";
 
-    public override IList<string> SearchPatterns => new List<string> { "pom.xml" };
+    public override IList<string> SearchPatterns => new List<string> { MavenManifest };
 
     public override IEnumerable<ComponentType> SupportedComponentTypes => new[] { ComponentType.Maven };
 
-    public override int Version => 3;
+    public override int Version => 4;
```

Comment:
No actual dependency graph changes but bumped it to identify when the fix was deployed, in case there is any unexpected regression.